components-core/src/main/java/com/adyen/checkout/components/api/LogoConnectionTask.kt

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2019 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by caiof on 11/3/2019.
+ */
+package com.adyen.checkout.components.api
+
+import android.graphics.drawable.BitmapDrawable
+import com.adyen.checkout.core.api.ConnectionTask
+import com.adyen.checkout.core.api.ThreadManager
+import com.adyen.checkout.core.log.LogUtil
+import com.adyen.checkout.core.log.Logger
+import java.util.ArrayList
+import java.util.concurrent.ExecutionException
+import java.util.concurrent.TimeUnit
+import java.util.concurrent.TimeoutException
+
+/**
+ * Task that wraps a Connection to get a Logo.
+ */
+class LogoConnectionTask(
+    private val logoApi: LogoApi,
+    val logoUrl: String,
+    callback: LogoCallback
+) : ConnectionTask<BitmapDrawable>(
+    LogoConnection(logoUrl)
+) {
+
+    var mCallbacks: MutableList<LogoCallback> = ArrayList()
```

Comment:
Since the amount of callbacks won't be too big maybe it doesn't matter too much, but I would make this a `HashSet` for a small optimization.